lib/graphql/schema/argument.rb

```diff
@@ -129,6 +130,17 @@ def description(text = nil)
         end
       end
 
+      attr_writer :comment
```

Comment:
Why the `attr_writer` here? Can it be removed, since people can use `comment(...)` instead?

Reply:
I mirrored how the description was made for argument. If it's not necessary, (I don't have such an insight) I can remove it.

Reply:
@rmosolgo Is it ok, if it stays the same as description then?

Reply:
Yes, seems alright, thanks!